src/screens/DoneTasksScreen/index.tsx

```diff
@@ -0,0 +1,50 @@
+import {LayoutView, MainView, WrapperButton, WrapperNotes} from './styled';
+import {BackStyle} from '@ui/BackStyle';
+import {Header} from '@components/Header';
+import {AddNoteButton} from '@ui/AddNoteButton';
+import {useCallback, useContext, useMemo, useState} from 'react';
+import {CreateNoteModal} from '@components/CreateNoteModal';
+import {ScrollView} from 'react-native-gesture-handler';
+
+import {Note} from '@ui/Note';
+import {NotesContext} from '@context/contextProvider';
+import {StyleSheet} from 'react-native';
+
+export function DoneTasksScreen() {
+  const [visible, setVisible] = useState(false);
+
+  const notes = useContext(NotesContext);
+  const doneNotes = useMemo(() => notes.filter(note => note.checked), [notes]);
+
```

Comment:
you always try to get some notes(done, daily, etc)
Maybe you can move this all logic to custom hook?